src/main/java/io/pravega/connectors/flink/serialization/PravegaDeserializationSchema.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *   
+ */
+package io.pravega.connectors.flink.serialization;
+
+import io.pravega.client.stream.Serializer;
+import java.io.IOException;
+import java.io.Serializable;
+import java.nio.ByteBuffer;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.streaming.util.serialization.AbstractDeserializationSchema;
+
+/**
+ * A deserialization schema adapter for a Pravega serializer.
+ */
+public class PravegaDeserializationSchema<T extends Serializable> extends AbstractDeserializationSchema<T> {
```

Comment:
There is nothing really specific to Pravega in this class, should we give it a more generic name? I'm also a bit confused by the use of the term `schema` here. Why are we calling it schema?